libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -238,6 +244,36 @@ data WithStatusBase (m :: Type -> Type) (cfg :: Type) = WithStatusBase
   }
   deriving stock (Generic, Typeable, Functor)
 
+--------------------------------------------------------------------------------
+-- DbFeature
+
+-- | Feature data stored in the database, as a function of its default values.
+newtype DbFeature cfg = DbFeature
+  {unDbFeature :: WithStatusNoLock cfg -> WithStatusNoLock cfg}
+
+instance Semigroup (DbFeature cfg) where
+  DbFeature f <> DbFeature g = DbFeature (f . g)
```

Comment:
are you sure you want `f` to write over `g`?

Reply:
I'm not sure which one is better. I actually think it would be more confusing to invert the order here.